src/plugins/cosmos/components/modals/Send/SendMaxButton/SendMaxButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@chakra-ui/react'
```

Comment:
Why does this deserve a new component ? Is it any different from the existing `SendMaxButton`?

https://github.com/shapeshift/web/blob/b8a7739d92ea13a4b1db9a7c53038fa2325c5c3a/src/components/Modals/Send/SendMaxButton/SendMaxButton.tsx